ReleaseTooling/Sources/FirebaseReleaser/InitializeRelease.swift

```diff
@@ -93,20 +93,24 @@ struct InitializeRelease {
         contents.insert(contentsOf: "'" + version + "'", at: versionStartIndex)
       } else {
         // Replace version in string like ss.dependency 'FirebaseCore', '6.3.0'
-        guard let range = contents.range(of: pod) else {
+        // Obtain all the ranges of `pod`'s occurences.
+        guard let ranges = contents.ranges(of: pod) else {
           // This pod is not a top-level Firebase pod dependency.
           continue
         }
-        var versionStartIndex = contents.index(range.upperBound, offsetBy: 2)
-        while !contents[versionStartIndex].isWholeNumber {
-          versionStartIndex = contents.index(versionStartIndex, offsetBy: 1)
-        }
-        var versionEndIndex = contents.index(versionStartIndex, offsetBy: 1)
-        while contents[versionEndIndex] != "'" {
-          versionEndIndex = contents.index(versionEndIndex, offsetBy: 1)
+
+        for range in ranges {
+          var versionStartIndex = contents.index(range.upperBound, offsetBy: 2)
```

Comment:
optional: I know, it was here before your change, but it would be useful to add either comments explaining the intent or refactor it to a method(s) to make it self-documentary or both. I don't feel too strong about it but I think it will help future us.

Reply:
SGTM. To simplify this method even more, I'm curious if it was ever considered to bump the versions by calling
`contents.replacingOccurrences(of: oldVersion, with: newVersion)`

Reply:
I guess we may not always now old version for each pod that's why we try to parse the version from the file first. I think @paulb777 should have more context on it as the original author.

Reply:
Yep, when it started, there weren't consistent oldVersions. Also, a generic string replace, in theory, could change the string in a name or comment incorrectly. 

However, it could be fine to simplify now since `oldVersion` should be able to be defined and the context check might be not worth the extra complexity.

Reply:
@paulb777 Makes sense. I was in the middle of refactoring some logic into a method when you replied so I figured I'd at least push those changes. WDYT of the current state? cc: @maksymmalyhin

Reply:
LGTM

Reply:
IMO the concerns about unintentional changes look still relevant. We may mitigate it by using e.g. regex, but I'm not sure if it will end up more readable eventually. The current state LGTM.